js/src/dom/event-handler.js

```diff
@@ -113,7 +114,7 @@ function bootstrapDelegationHandler(element, selector, fn) {
 
           if (handler.oneOff) {
             // eslint-disable-next-line unicorn/consistent-destructuring
-            EventHandler.off(element, event.type, fn)
+            EventHandler.off(element, event.type, selector, fn)
```

Comment:
Note to reviewers, this fixes a bug with the one time delegated event handlers. Added a test to the specs too.